etc/completions

```diff
@@ -1,5 +1,807 @@
-$command $opt* ($revision|$path|$remote)* --? $path+
+(add|stage) $opt* --? $path+
+  --all
+  --dry-run
+  --edit
+  --force
+  --ignore-errors
+  --ignore-missing
+  --ignore-removal
+  --intent-to-add
+  --interactive
+  --no-all
+  --no-ignore-removal
+  --refresh
+  --update
+  --verbose
+
+blame $opt* $revision? --? $path
+  --incremental
+  --line-porcelain
+  --no-progress
+  --porcelain
+  --progress
+  --root
+  --score-debug
+  --show-email
+  --show-name
+  --show-number
+  --show-stats
+  --abbrev $anything
+  --contents $path
+  --date $anything
+  --encoding $anything
+  --reverse $revision
+  --since $anything
+  -S $path
+
+# Listing
```

Comment:
`# Branch listing`? 

Also, is this a YAML file? If so, would using indentation help convey the hierarchy more clearly?